src/theme/material/toolbar.m.css

```diff
@@ -1,2 +1,10 @@
+@import './variables.css';
+
 .root {
+	background-color: var(--mdc-theme-primary);
```

Comment:
i think there may be some sizing etc to take from the material theme for toolbar here?